test/controllers/brands/collaborations_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```

Comment:
I think you could remove these test files 🤔

Reply:
The whole test folder?

Reply:
@Timbit87 ah they seem to be auto generated. You can leave them in as they wont cause any problems. Sorry for the confusion and probably just a nitpicky comment 🙏